internal/featuresupport/feature_support_level.go

```diff
@@ -104,10 +101,14 @@ func GetFeatureSupportList(openshiftVersion string, cpuArchitecture *string, pla
 
 // IsFeatureAvailable Get the support level of a given feature, cpuArchitecture is optional
 // with default value of x86_64
-func IsFeatureAvailable(featureId models.FeatureSupportLevelID, openshiftVersion string, cpuArchitecture *string) bool {
+func IsFeatureAvailable(featureId models.FeatureSupportLevelID, openshiftVersion string, cpuArchitecture *string, highAvailabilityMode *string) bool {
+	if highAvailabilityMode == nil {
```

Comment:
in which case we will get highAvailabilityMode == nil?

Reply:
if not configured, mostly in API call for featuresupport - but also subsystem test and unit tests